src/open_inwoner/templates/pages/sitemap/sitemap.html

```diff
@@ -0,0 +1,60 @@
+{% extends "master.html" %}
+{% load i18n list_tags %}
+
+{% block content %}
+    <h1 class="utrecht-heading-1">{% trans "Links naar pagina's op het Open Inwoner Platform" %}</h1>
+
+    <div class="grid sitemap">
+        {# onderwerpen #}
+        <section class="sitemap__section">
+            <a class="utrecht-heading-2" href="{% url 'products:category_list' %}">{% trans "Onderwerpen" %}</a>
+            <ul class="sitemap__list">
+                {% for node in category_nodes %}
+                    {% include "pages/sitemap/category_item.html" with node=node %}
+                {% endfor %}
+            </ul>
+        </section>
+
+        {% if request.user.is_authenticated %}
+            <section class="sitemap__section">
+                {# profile pages heading #}
+                {% for page in cms_profile_pages|slice:":1" %}
+                    {% with link=page.url_name text=page.link_text %}
+                        <a class="utrecht-heading-2" href="{% url link %}">{% trans text %}</a>
+                    {% endwith %}
+                {% endfor %}
+                <ul class="sitemap__list">
```

Comment:
Since we need to start using NL-Design_System components, I don't think we should create new ones. But just keep these for now.

I'm thinking of directly using the nice LinkList https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css_css-link-list--docs

Of for simplicity just these: https://nl-design-system.github.io/utrecht/storybook/?path=/story/css_css-unordered-list--nested

So that's what I'll be doing later.